features/support/remote_helpers.rb

```diff
@@ -12,13 +12,13 @@ def base_url domain
 
 
 # Returns the remote URL for a new pull request for the given domain and branch
-def pull_request_url domain, branch_name, repo
+def pull_request_url domain, branch, parent_branch, repo
   case domain
   when 'Bitbucket'
     sha = recent_commit_shas(1).join('')[0, 12]
-    "https://bitbucket.org/#{repo}/pull-request/new?source=#{CGI.escape repo}%3A#{sha}%3A#{branch_name}"
+    "https://bitbucket.org/#{repo}/pull-request/new?source=#{CGI.escape repo}%3A#{sha}%3A#{branch}"
```

Comment:
This hasn't been updated to include the parent branch. Looks like we need to add a test for bitbucket with a parent branch